src/homeworks/test_32/index.ts

```diff
@@ -8,3 +10,5 @@
  * @argument data - { name: string; age: number }[]
  * @returns data - { name: string; age: number, position: number }[]
  */
+export const getDataWithPosition = (data: Data[]): getDataAndPos[] =>
+  data.map(({ name, age }, index) => ({ name, age, position: index + 1 }));
```

Comment:
Non prendiamo le proprieta' uno alla volta. Usiamo lo spread operator.
```suggestion
  dataList.map((data, index) => ({ ...data, position: index + 1 }));
```